### PR TITLE
Adds eslint support, switches tabs to spaces

### PR DESCRIPTION
**Pull request by swbloom**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            2
```
Comment (Rchristiani) [inline .eslintrc.js]: 2?!?! I am a 4 man myself....

**Review by Rchristiani — CHANGES_REQUESTED** [review]
Thoughts?!?!!

File: .eslintrc.js (hunk 1 above)
Comment (swbloom) [reply .eslintrc.js]: darcy made me change it to 2

Comment (swbloom) [reply .eslintrc.js]: i like 4 better as well

**Comment by swbloom** [comment]
updated!

**Pull request merged**